types/react-alice-carousel/index.d.ts

```diff
@@ -7,115 +7,38 @@
 import * as React from "react";
 
 export interface EventObject {
-	item: number;
-	slide: number;
+  item: number;
+  slide: number;
+  itemsInSlide: number;
 }
 
 export interface Props {
-	/**
-	 * Fired when the event object is changing / returns event object
```

Comment:
Is there a particular reason you deleted all of this documentation?

Reply:
I guess comments are redundant. Full description is in the [doc](https://github.com/maxmarinich/react-alice-carousel#props).

Reply:
I would suggest leaving the JSDoc comments - that way, editors can display the documentation inline.